pkg/ddc/alluxio/shutdown.go

```diff
@@ -203,6 +200,12 @@ func (e *AlluxioEngine) destroyWorkers(workers int32) (err error) {
 			continue
 		}
 
+		for _, label := range toUpdate.Labels {
+			if label == labelExclusiveName {
```

Comment:
Since `toUpdate.Labels` is of type map[string]sting, 
1. `label` is a value while `labelExclusiveName` is used as a key here. They will never equal to each other.
2. I think it will be more efficient if it's implemented like this:
```
exclusiveLabelValue := e.namespace + "-" + e.name
if val, exist := toUpdate.Labels[labelExclusiveName]; exist && val == exclusiveLabelValue {
    delete(toUpdate.Labels, labelExclusiveName)
}
```

```

Reply:
Yep, i want compare the key, thanks a lot for your detailed review and good suggestion. @TrafalgarZZZ